atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetrics.java

```diff
@@ -417,7 +417,7 @@ public interface MetricsConfiguration {
          */
         @Value.Default
         default Set<SweeperStrategy> trackedSweeperStrategies() {
-            return ImmutableSet.copyOf(SweeperStrategy.values());
+            return ImmutableSet.of(SweeperStrategy.CONSERVATIVE, SweeperStrategy.THOROUGH);
```

Comment:
nit: Can we go the other way? explicitly remove non sweepable from the set rather than constructing it from the two valid elements? It's not a big deal hence the nit, but I imagine someone one day will forget this is a thing and wonder why there's no metrics for their new sweeper strategy